build-system/pr-check/build-targets.js

```diff
@@ -232,7 +232,16 @@ const targetMatchers = {
     );
   },
   [Targets.LINT_RULES]: (file) => {
-    return file.endsWith('.eslintrc.js') || file == 'package.json';
+    if (isOwnersFile(file)) {
+      return false;
+    }
+    return (
+      file.startsWith('build-system/eslint-rules') ||
+      file.endsWith('.eslintrc.js') ||
+      file.endsWith('.prettierrc') ||
```

Comment:
Suggest using exact name since there's only one `.prettierrc` in the repo.
```suggestion
      file == '.prettierrc' ||
```

Also add `.prettierignore` and `.eslintignore` to the list.